web/packages/hovercards/src/core.ts

```diff
@@ -278,13 +314,48 @@ export default class Hovercards {
 			}, [] )
 			.join( '' );
 
-		const jobInfo = [ jobTitle, company ].filter( Boolean ).join( ', ' );
+		let ctaButtons = '';
+		let contactsDrawer = '';
+		let sendMoneyDrawer = '';
+		const contactsDrawerId = 'contact-drawer';
+		const sendMoneyDrawerId = 'send-money-drawer';
+
+		if ( nonEmptyContacts.length || hasPayments ) {
+			if ( nonEmptyContacts.length ) {
+				ctaButtons += `
+					<button id="contact-btn" class="gravatar-hovercard__button">${ __( i18n, 'Contact' ) }</button>
+				`;
+
+				contactsDrawer = Hovercards._createDrawer(
+					contactsDrawerId,
+					__( i18n, 'Contacts' ),
+					Hovercards._createContactDrawerContent( nonEmptyContacts )
+				);
+			}
+
+			if ( hasPayments ) {
+				ctaButtons += `
+					<button id="send-money-btn" class="gravatar-hovercard__button">${ __( i18n, 'Send money' ) }</button>
+				`;
+
+				sendMoneyDrawer = Hovercards._createDrawer(
+					sendMoneyDrawerId,
+					__( i18n, 'Send money' ),
+					Hovercards._createSendMoneyDrawerContent( payments )
+				);
+			}
+
+			ctaButtons = `
+				<div class="gravatar-hovercard__buttons">${ ctaButtons }</div>
+			`;
+		}
 
 		hovercard.innerHTML = `
-			<div class="gravatar-hovercard__inner">
+			<div id="g-${ hash }" class="gravatar-hovercard__inner">
```

Comment:
Q: What is the `id` used for?

Reply:
That was a left over from when I tried to address the CSP issue. Good catch!